### PR TITLE
Update README with AutoValue and Kotlin examples

### PR DESCRIPTION
**Pull request by geralt-encore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,6 +189,33 @@ public class Tweet {
 }
 ```
 
+[`AutoValue`](https://github.com/google/auto/blob/master/value/userguide/index.md):
+```java
+@AutoValue
+@StorIOSQLiteType(table = "tweets")
+public abstract class Tweet {
+
+	// annotated methods should have package-level or public visibility
```
Comment (artem-zinnatullin) [inline README.md]: http://nedbatchelder.com/blog/201401/comments_should_be_sentences.html :)

File: README.md [hunk 2]
```diff
@@ -180,6 +189,33 @@ public class Tweet {
...
+	@StorIOSQLiteColumn(name = "author")
+	abstract String author();
+
+	@StorIOSQLiteColumn(name = "content")
+	abstract String content();
+
+    // parameters order depends on declaration order
```
Comment (artem-zinnatullin) [inline README.md]: same as above

File: README.md [hunk 3]
```diff
@@ -154,32 +154,68 @@ To **save you from coding boilerplate classes** we created **Annotation Processo
 
 *Notice that annotation processors are not part of the library core, you can work with StorIO without them, we just made them to save you from boilerplate*.
 
+`StorIOSQLite`:
 ```groovy
 dependencies {
-	// At the moment there is annotation processor only for StorIOSQLite
```
Comment (nikitin-da) [inline README.md]: 🙈

File: README.md [hunk 4]
```diff
@@ -154,32 +154,68 @@ To **save you from coding boilerplate classes** we created **Annotation Processo
...
 
 	// We recommend to use Android Gradle Apt plugin: https://bitbucket.org/hvisser/android-apt
 	apt 'com.pushtorefresh.storio:sqlite-annotations-processor:insert-latest-version-here'
 }
 ```
 
+`StorIOContentResolver`:
+```groovy
+dependencies {
+	compile 'com.pushtorefresh.storio:sqlite-annotations:insert-latest-version-here'
```
Comment (nikitin-da) [inline README.md]: `storio-content-resolver-annotations` and `storio-content-resolver-annotations-processor`

**Comment by nikitin-da** [comment]
Thanks, @geralt-encore!

**Pull request merged**